L/libblastrampoline/build_tarballs.jl

```diff
@@ -34,7 +34,7 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6",
+               julia_compat="1.8",
                init_block = """
                @static if VERSION < v"1.7.0-DEV.641"
```

Comment:
Then I guess we can get rid of this `init_block` too?

Reply:
Good point!